README.md

```diff
@@ -1,4 +1,51 @@
-# Python boilerplate for GitHub tasks
+# Token Authentication API
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+- Download [ModHeader](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj?hl=en)
+
+### In this task you will add the functionality of token authentication
+
+At this part of the task, we will do authorization by using tokens. The functionality of regular users will be limited so that they cannot add, delete or update other data on the sites, besides their orders.  Moreover, only authenticated users will be able to create an order.  Deletion will be prohibited even for the administrator, if only through the admin panel. That's because of  when we're deleting, for example, a genre, the other relationships from other tables won't deleted
```

Comment:
```suggestion
At this part of the task, we will do authorization by using tokens. The functionality of regular users will be limited so that they cannot add, delete or update other data on the site, besides their orders.  Moreover, only authenticated users will be able to create an order.  Deletion will be prohibited even for the administrator, if only through the admin panel. That's because of  when we're deleting, for example, a genre, the other relationships from other tables won't be deleted
```